WhaleSpotting/Repositories/LikeRepo.cs

```diff
@@ -0,0 +1,30 @@
+using WhaleSpotting.Models.Database;
+using WhaleSpotting.Models.Request;
+
+namespace WhaleSpotting.Repositories;
+
+public interface ILikeRepo
+{
+    public void Create(LikeRequest newLikeRequest);
+}
+
+public class LikeRepo : ILikeRepo
+{
+    private readonly WhaleSpottingDbContext context;
+
+    public LikeRepo(WhaleSpottingDbContext context)
+    {
+        this.context = context;
+    }
+    
+    public void Create(LikeRequest newLikeRequest)
+    {
+        var insertResult = context.Likes.Add(new Like
```

Comment:
We don't use this `insertResult` variable anywhere, do we want to return it? If not we don't need to set it as a variable